src/Testcontainers.Bigtable/BigtableContainer.cs

```diff
@@ -0,0 +1,15 @@
+namespace Testcontainers.Bigtable;
+
+/// <inheritdoc cref="DockerContainer"/>
+[PublicAPI]
+public class BigtableContainer: DockerContainer
+{
+  public readonly string ProjectId = "project-id";
```

Comment:
Not sure if this is good practice in this repo. Not sure if the specific project-/instance-id values are required by the emulator or not.

Reply:
I need to take a closer look at it. AFAIK, we do not have something like this in any of the other Google Cloud modules. We simply create the values in the [tests](https://github.com/testcontainers/testcontainers-dotnet/blob/46c05d7abdcc92bf468947ea6b2826dfb36b1908/tests/Testcontainers.PubSub.Tests/PubSubContainerTests.cs#L19). Maybe that will help already.

Reply:
I think that sounds like a fairt option, looks like the Java implementation does not expose these, so removed them to make it more equal to the BigtableEmulatorContainer in java gcloud module.